### DISCUSSION TITLE
拉格朗日點?

### DISCUSSION THREAD

**Discussion by xCk27x** [1101-GS4719-C/Python-Community-GS4719-C#239] [科技新知]
在12/25號的聖誕節那天 NASA送了天文學家-或是全人類一個禮物，**詹姆斯韋伯太空望遠鏡**
他將會飛到距離地球160萬公里的第二拉格朗日點上 預計在2022年的暑假會傳回第一波照片

![Lagrangianpointsanimated](https://user-images.githubusercontent.com/90547641/147633424-ecb9d765-6ed2-4581-b5bf-3b9cc6e9078b.gif)
圖中標示 2 的那個點就是第二拉格朗日點

在拉格朗日點上，物體相對於地球跟太陽是靜止的，所以韋伯可以不用頻繁地進行位置修正，以更穩定的進行觀測。

但將衛星放在拉格朗日點也不是沒有壞處的，最明顯的缺點就是「**無法維修**」，這也是為什麼韋伯會從原先預定發射的2007年一直推遲到2021/12/25的主要原因。因為一旦有任何儀器出了差錯，都將是永久性的。

**Comment by enoch-lightning** [comment]
這看起來很厲害，所以太空望遠鏡裡面也有寫程式嗎？

**Reply by xCk27x** [reply]
一定是有的，一部分是裡面的科學儀器都非常的複雜，如果想要對這些儀器進行操作，就免不了程式的使用。另一方面是韋伯望遠鏡的非常的巨大，他的望遠鏡直徑為6.5公尺，遮光罩直徑更長達21公尺，已經遠超火箭所能乘載的大小了，所以韋伯望遠鏡是需要經過多次摺疊的，在展開的過程中也會用到許多程式來展開望遠鏡，並且判斷各個儀器的展開是否順利。

![OIP (2)](https://user-images.githubusercontent.com/90547641/147672517-fe63a9dd-f1e5-4611-90a0-91260fc79889.jpg)

**Reply by enoch-lightning** [reply]
或許LabVIEW在這裡就派上用場了。
